scripts/ci_check_no_file_changes.sh

```diff
@@ -34,7 +34,7 @@ REGEXES=()
 for CHECK in "$@"
 do
   if [[ ${CHECK} == "python" ]]; then
-    REGEX="(^\.github\/workflows\/.*python|^tests\/|^superset\/|^setup\.py|^requirements\/.+\.txt)"
+    REGEX="(^\.github\/workflows\/.*python|^tests\/|^superset\/|^setup\.py|^requirements\/.+\.txt|^\.pylintrc)"
```

Comment:
This maybe overkill, i.e., enable all Python checks on only a change to the `.pylintrc` file, though changes to only the `.pylintrc` file are rare.

Reply:
I agree with this; these are so uncommon doing a bit of overkill here is better than not doing it at all.